src/components/categories/Categories.jsx

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import React from 'react'
+import { Link } from 'react-router-dom';
+import { useState,useEffect } from 'react';
```

Comment:
So line number 2 and 4 can be combined here and written as import {React,useState,useEffect} from "react"

Reply:
Okay will do all things